### ISSUE TITLE
Model performance

### ISSUE THREAD

**Issue by guozhiyao** [benjs/nfnets_pytorch#6]
Hi, is the `Top-1 accuracy this implementation` reported in the README the result of your code training from scratch?

**Comment by benjs** [comment]
The accuracy is the result of taking the pretrained weights from the original repo and running `eval.py`.

When training from scratch using this code, there are multiple things missing for best results.
`train.py` should reach the baseline accuracy as described in the paper, however, this is not tested as I don't have the computing resources or money to train such large models.

**Comment by guozhiyao** [comment]
Then why the results are inconsistent with `Top-1 accuracy Brock et al.`?

**Comment by benjs** [comment]
They preprocess the input images differently. I tried to implement their method but it was a bit worse than simple resizing.

With such small differences, there can be many things that cause them (resizing algorithm, ...)

**Comment by guozhiyao** [comment]
ok. Thanks a lot.

**Comment by guozhiyao** [comment]
I calculated the #Params and found that they were inconsistent with those reported in the paper. The calculation method
```
size = 0
for p in net.parameters():
    size += p.numel()
print(size*4/1024/1024)
```

**Comment by benjs** [comment]
What was the result of your calculation?

**Comment by guozhiyao** [comment]
I test all NFNets and set the `num_classes=1000`, `stochdepth_rate=0.25`, `alpha=0.2`, `se_ratio=0.5`, `activation='gelu'`, and the results are: 272.7MB, 506.0MB, 739.2MB, 972.5MB, 1205.7MB, 1439.0MB, 1672.2MB, 1905.5MB.

**Comment by benjs** [comment]
I guess the correct parameter count is in the `#Params` column in table 3 of the paper.
 
Your code counts the number of bytes, assuming that one parameter is of type float32 and thus has 4 bytes.

The paper however simply provides the number of parameter, not the number of bytes.

When you change your print statement to
`print(size/1000/1000)`, you'll see that the count is correct.

**Comment by guozhiyao** [comment]
Yes, you are right. Thank you for your reply.
